source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+    "fields_title", "A title for the fields.", "string", "No"
+    "transitions", "A list of transitions to include in the message. When the user clicks on one of the transitions, the workflow transitions to the specified step.", "[]Button", "No"
+    "transition_title", "A title for the transitions.", "string", "No"
+
+Below is an example JSON for a step containing one ``post`` start action, which the message below to a channel created by the ``CreateWarroom`` action.
+
+.. image:: ../images/incident-response-app-post-action.png
+   :alt: Incident Response App: Post Action
+
+.. code-block:: json
+
+  "steps": [
+      {
+          "name": "Triage",
+          "start_actions": [
+              {
+                  "name": "TriagePost",
+                  "type": "post",
+                  "channel_name": "{{.Action.CreateWarroom.ChannelName}}",
+                  "message": "New issue to triage:\n ```{{.Trigger.IssueReported.Message}}```",
+                  "fields_title": "Please add details to this issue",
+                  "fields": [
+                      {
+                          "name": "Escalate",
+                          "type": "button",
+                          "description": "Escalate to Team Lead for immediate action"
+                      },
+                      {
+                          "name": "Impact",
+                          "type": "options",
+                          "description": "The impact of the incident on an individual user",
+                          "options": [
+                              "I1",
+                              "I2",
+                              "I3"
+                          ]
+                      },
+                      {
+                          "name": "Reach",
+                          "type": "options",
+                          "description": "The number of users impacted",
+                          "options": [
+                              "10,000+",
+                              "1,000-10,000",
+                              "Less than 1,000"
+                          ]
+                      }
+                  ],
+                  "transitions_title": "Transition to step",
+                  "transitions": [
+                      {
+                          "label": "Triaged",
+                          "description": "Issue has been triaged",
+                          "to": "DevelopFix"
+                      },
+                      {
+                          "label": "Resolved",
+                          "description": "Issue has been resolved",
+                          "to": "Resolved"
+                      }
+                  ]
+              }
+          ]
+      }
+  ]
+
+.. tip::
+  Note that the above JSON uses ``{{.Action.CreateWarroom.ChannelName}}`` as the channel name. These are template variables which allow you to dynamically specify parameters based on other actions or steps within the app workflow schema.
```

Comment:
It might be helpful to have a section specifically for template variables.  You reference them a couple of times throughout, but it might be useful to have it all in one section, where you can define the purpose overall of these and specify each one and what you can do with it.